### DISCUSSION TITLE
Welcome to pymdown-extensions Discussions!

### DISCUSSION THREAD

**Discussion by facelessuser** [facelessuser/pymdown-extensions#1173] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Andre601** [comment]
Heya.

I'm Andre (Often known as Andre_601).
I use PyMdown alongside MkDocs and Materials for MkDocs to build documentation and pages for various things. The various features this extension offers help me a lot throughout making the docs, so I'm really thankful for the work put into it.

Some projects I use PyMdown in:

- https://andre601.ch/GitHub-Markdown
- https://docs.purrbot.site

**Reply by facelessuser** [reply]
You can share links to live examples. As long as the content is PG and generally non-offensive, I think examples are fine.
